### PR TITLE
Leaf/f3.1 logged in home screen

### PR DESCRIPTION
**Pull request by tdang2180**
#134 
Created the logged-in home screen
- Created a message footer
- Fixed Header theme

### CODE REVIEW

File: meshapp/src/components/Header.tsx [hunk 1]
```diff
@@ -7,11 +7,15 @@ import React from "react";
...
+    <AppBar
+      position="static"
+      sx={{ backgroundColor: "primary.main" }}
+      elevation={0}
+    >
       <Toolbar>
         {/*---Left Side Menu Elements (Menu/Notification Icon)---*/}
         <Grid container>
-          <IconButton sx={{ mr: 2, color: "#0000008F" }}>
+          <IconButton sx={{ mr: 2, color: "secondary.main" }}>
```
Comment (Darren52BT) [inline meshapp/src/components/Header.tsx:18]: the color should be "action.active", you can see the difference if you switch to light mode

File: meshapp/src/components/Header.tsx [hunk 2]
```diff
@@ -20,7 +24,7 @@ const Header = () => {
             edge="start"
             color="inherit"
             aria-label="menu"
-            sx={{ mr: 2, color: "#0000008F" }}
+            sx={{ mr: 2, color: "secondary.main" }}
```
Comment (Darren52BT) [inline meshapp/src/components/Header.tsx:27]: same issue as the other icon button

**Review by Darren52BT — CHANGES_REQUESTED** [review]
Aside from the button color, everything looks good for the most part. There is also the issue of all the sizes becoming way too big after we go below 600px. I dont think we're too concerned about styles if the screen width is super small, but I would suggest maybe extending those sizes for a minimum screen width of 600px to screen widths below 600px as well. It might start breaking down again at around 400px but at that point I don't think that's a big deal, we can go back to fix it if we need to.

File: meshapp/src/components/Header.tsx (hunk 1 above)
Comment (tdang2180) [reply meshapp/src/components/Header.tsx]: 👍

**Review by Darren52BT — APPROVED** [review]
lgtm

**Pull request merged**